packages/prices/utils/price.ts

```diff
@@ -113,6 +115,50 @@ export const getCurrency = (
 	};
 };
 
+/**
+ * Get the integer value from the decimal price.
+ *
+ * @param {number} priceInt The price in decimals.
```

Comment:
What does "The price in decimals" mean? It is unclear to me what's being passed in.

Reply:
I changed `The price in decimals.` to `The price in minor unit value, e.g. 100 for $1.00.` to ensure that it's clear what is passed to the function.